draft-ietf-masque-h3-datagram.md

```diff
@@ -90,86 +79,16 @@ when, and only when, they appear in all capitals, as shown here.
 
 # Multiplexing
```

Comment:
Strictly editorial: a better way to structure this section would be to describe what we defin multiplexing is. Then the follow on paragraphs state how each version can (or cannot) satisfy that. That would mean simplifying paragraph 2 of this section by extracting the "need the ability" piece so that the text focuses on how not why.

Such an editorial change could be tracked as follow on work, since this PR is focuesd on design changes.

Reply:
This sounds good, but I agree - let's tackle this as editors once the outcome of the design team is merged.